src/main/resources/views/CommercialVehicles/VTG30.hbs

```diff
@@ -148,6 +148,27 @@
                         </div>
                         <!-- /heading-info -->
                     </div>
+                    <div class="header__three-columns-three">
+                        <!-- heading-info -->
+                        <div class="heading-info">
+                            <h2 class="heading-info__title">
+                              <span class="heading-info__title-left">
+                              <span class="circle-text">
+                              <span class="circle-text__number">(2c)</span>
+                              </span>
+                              </span>
+                              <span class="heading-info__title-right">
+                              <span class="heading-info__title-message">
+                              Registration Number
+                              </span>
+                              </span>
+                            </h2>
+                           <span class="heading-info__content" id="trn">
+                               {{failData.trn}}
```

Comment:
To check with Olivia, Umer and I need clarification on the following:
- In case of HGV do we hide the Trn (VTG30 only)
- In case of empty do we show the empty field or add a - (VTG5A as well)

Reply:
conditional logic added to handle the same template for different vehicle types.